### DISCUSSION TITLE
第4节课作业(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#348] [作业]
> 1. 基础作业：
> 
> 构建数据集，使用 XTuner 微调 InternLM-Chat-7B 模型, 让模型学习到它是你的智能小助手，效果如下图所示，本作业训练出来的模型的输出需要将不要葱姜蒜大佬替换成自己名字或昵称！
> 
> 2. 进阶作业：
> 
> 将训练好的Adapter模型权重上传到 OpenXLab、Hugging Face 或者 MoelScope 任一一平台。 将训练好后的模型应用部署到 OpenXLab 平台，参考部署文档请访问：https://aicarrier.feishu.cn/docx/MQH6dygcKolG37x0ekcc4oZhnCe
> 
> **整体实训营项目：**
> 
> 时间周期：即日起致课程结束
> 
> 即日开始可以在班级群中随机组队完成一个大作业项目，一些可提供的选题如下：
> 
> 人情世故大模型：一个帮助用户撰写新年祝福文案的人情事故大模型 中小学数学大模型：一个拥有一定数学解题能力的大模型 心理大模型：一个治愈的心理大模型 工具调用类项目：结合 Lagent 构建数据集训练 InternLM 模型，支持对 MMYOLO 等工具的调用 其他基于书生·浦语工具链的小项目都在范围内，欢迎大家充分发挥想象力。
> 
> 作业[参考答案](https://github.com/InternLM/tutorial/blob/main/xtuner/self.md)

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec4

**Comment by cemcke** [comment]
http://t.csdnimg.cn/6DuIU

**Comment by admlrz** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135648475

**Comment by SChaoWei** [comment]
https://blog.csdn.net/qq_45919032/article/details/135673200?spm=1001.2014.3001.5501

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135660213

**Comment by JokerOfRevenge** [comment]
https://www.processon.com/v/65a9fa686ccfdd2c2cd03a56

**Comment by dearSocrates** [comment]
https://editor.csdn.net/md/?articleId=135511248

**Comment by MIracleyin** [comment]
https://nnkwjqimr7.feishu.cn/docx/BlnOd3u4soGLcgxVZkZcnHAtnnf?from=from_copylink

**Comment by Tian-CASIA** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135554607
